src/Microsoft.Azure.WebJobs.Host/Scale/ScaleMonitorService.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Hosting;
+using System;
```

Comment:
Same here - you're messing up the using orderings randomly in files you're moving over. Update your VS settings to order system usings first